.github/workflows/build.yml

```diff
@@ -270,6 +279,42 @@ jobs:
           rm -rf temp
           mv signed temp
 
+      - name: Generate VSA
+        if: env.shouldSign == 'true'
+        env:
+          AWS_KMS_REGION:  ${{ job.build.steps.auth.with.args.aws-region }}
+          AWS_KMS_KEY_ARN: ${{ job.build.steps.sign.env.AWS_KMS_KEY_ARN }}
+        run: |
+          args+=(
+            --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
+
+            # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
+            "docker/image-signer-verifier:$BASHBREW_IMAGE_SIGNER"
+
+            verify
+
+            -D
+
+            --platform="$BASHBREW_ARCH"
+            --vsa=true
+
+            --tuf-source=oci
+            --tuf-meta-source="$BASHBREW_TUF_META"
+            --tuf-targets-source="$BASHBREW_TUF_TARGETS"
+
+            --policy-id="$BASHBREW_SIGNING_POLICY"
+
+            --kms="AWS"
+            --kms-region="$AWS_KMS_REGION"
+            --kms-key-ref="$AWS_KMS_KEY_ARN"
+
+            --image=oci:///doi-build/signed
+            --referrers-source="$BASHBREW_REFERRERS_REPO"
+            --referrers-dest="$BASHBREW_REFERRERS_REPO"
```

Comment:
When specified this way, is this tool pulling, combining, and pushing the referrers objects itself directly, or is this expected to somehow be part of the OCI bundles we're passing around?

Reply:
It is pushing the referrers manifests as it signs the attestations. 

We technically don't need to pass around the oci layout unless we were attaching attestations (`--attach=true`) to the index or removing the unsigned statements (`--keep=false`). 

We already need to refactor some things to get this working, so I'll see if we can make the outputs optional. One plus to passing around the oci layout, is that if we ever decide to remove the unsigned statements, it would be a matter of setting `--keep=false`.